### PR TITLE
New changes in the server and the back end

### PR DESCRIPTION
**Pull request by Agaba-derrick**

### CODE REVIEW

**Comment by Agaba-derrick** [comment]
hey @danielmarv tried to push some changes in the server hope you review with me and make some changes where necessary

File: server/models/customer.model.js
Comment (Agaba-derrick) [inline server/models/customer.model.js:1]: hey @danielmarv am currently working on this one

File: server/.env
Comment (Agaba-derrick) [inline server/.env:1]: this is a file .env that simply creates a script for database

Comment (Agaba-derrick) [inline server/.env:1]: i also found out that .jsn file was missing and had it figured out here it is pending for review

File: server/server.js
Comment (Agaba-derrick) [inline server/server.js:1]: will leave this modul for you to review @danielmarv

File: server/models/customer.model.js
Comment (danielmarv) [reply server/models/customer.model.js:1]: Ok man i was away buh am back

Comment (danielmarv) [reply server/models/customer.model.js:1]: And on the server i thought we r to use MongoDB buh if its the best way lets use that 
BTW lets make the server.js as the index.js

**Comment by danielmarv** [comment]
ok now am to make some changes

**Pull request merged**